c_glib/arrow-glib/basic-array.cpp

```diff
@@ -544,6 +544,30 @@ garrow_array_view(GArrowArray *array,
   }
 }
 
+/**
+ * garrow_array_diff:
```

Comment:
Could you rename diff to `garrow_array_diff_unified`?
We'll provide low-level `garrow_array_diff` API later.

Reply:
I've renamed it.